public/addEdit.js

```diff
@@ -0,0 +1,142 @@
+import { enableInput, inputEnabled, message, setDiv, token } from "./index.js";
+import { showJobs } from "./jobs.js";
+
+const addEditDiv = document.getElementById("edit-job");
+const company = document.getElementById("company");
+const position = document.getElementById("position");
+const status = document.getElementById("status");
+const addingJob = document.getElementById("adding-job");
+
+export const handleAddEdit = () => {
+  const form = document.querySelector("#formSubmit");
```

Comment:
Nitpick:
This `form` variable can also reside outside of the `handleAddEdit` function. Also, I’d recommend using a more unique I’d than `formSubmit`. Something like `addEditJobForm` is more descriptive.